govuk-one-login/authentication-frontend
# AUT-2685: reducred timeout for sms opt lockouts

## What

Create Account/Account Recovery - lockout duration for > 5 incorrect SMS codes should remain at 15 mins

## How to review

1. Code Review
2. Update sandpit terraform variable _ reduced_code_block_duration_minutes_ to a reduced duration from _code_entered_wrong_blocked_minutes_
3. Deploy to sandpit with `./deploy-sandpit.sh -a`
4. Enter incorrect sms opt code in account creation and account recovery journeys
5. Ensure lockout is as expected
